### PR TITLE
Made delete_fsxn_svm smarter

### PR DESCRIPTION
**Pull request by kcantrel**
Made all the create and delete script support a 'wait' option; Made the delete_fsxn_svm support deleting all the volumes under the SVM; Now delete_fsxn_filesystem calls delete_fsxn_svm to delete SVMs, and delete_fsxn_svm calls delete_fsxn_volume to delete volumes.

### CODE REVIEW

File: Management-Utilities/fsx-ontap-aws-cli-scripts/README.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ Before running the UNIX based scripts, make sure the following package is instal
 
 | Script                  | Description     |
 |:------------------------|:----------------|
-|create_fsxn_filesystem   | Creates a new FSx for NetApp ONTAP file-system |
-|create_fsxn_svm          | Creates a new Storage Virtual Server (svm) in a soecific FSx ONTAP filesystem |
+|create_fsxn_filesystem   | Creates a new FSx for NetApp ONTAP file-system. |
+|create_fsxn_svm          | Creates a new Storage Virtual Server (SVM) in a soecific FSx ONTAP filesystem. |
```
Comment (nichollri) [inline Management-Utilities/fsx-ontap-aws-cli-scripts/README.md:18]: spelling, should be "in a specific FSx ONTAP"

**Review by nichollri — COMMENTED** [review]
Is there a reason why on fsxn-create you made the option --wait, but on svm create you made it just -w?

**Comment by kcantrel** [comment]
> Is there a reason why on fsxn-create you made the option --wait, but on svm create you made it just -w?

Since creating a file system requires a lot of options, I had to go with the double dash "long name" method of passing options, whereas with creating a SVM it only requires two, and only accepts 5 options. I didn't want to change all the other scripts to conform to what I had to do with create_fsxn_filesystem.

I could have made the -w option consistent among all the scripts. but I decided to make all the options for the create_fsxn_filesystem script require the double dash method so it would have a consistent semantics. I did make one exception in that it recognizes -h to display help. Having said that, it, of course it wouldn't take much to add -w as another way to specify the wait option if you really think it is needed.

**Review by nichollri — COMMENTED** [review]
I think it is fine.  Let's just get this merged in.

**Review by nichollri — APPROVED** [review]

**Review by nichollri — APPROVED** [review]
Tested out latest changes for error messages and they displayed correctly when I had snapmirrors and vserver peers on cleanup.

**Pull request merged**